packages/app-store/closecom/components/EventTypeAppCardInterface.tsx

```diff
@@ -10,6 +10,11 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
 
   const { enabled, updateEnabled } = useIsAppEnabled(app);
 
+  // CRM backwards compatibility
+  if (enabled === undefined) {
+    updateEnabled(true);
+  }
```

Comment:
For CRM apps not in the event type metadata, we assume that they're credentials created before this PR hits production. By default they are enabled automatically. If a user goes to the event type page we should indicate that to the user. Also by taking this approach it causes a dirty field so if the user does hit save it'll create the app data in the metadata.